examples/misc/bin/try_dart/functions.dart

```diff
@@ -7,9 +7,9 @@ int timesTwo(int x) {
 int timesFour(int x) => timesTwo(timesTwo(x));
 
 // Functions are objects.
-int runTwice(int x, Function f) {
+int runTwice(int x, int Function(int) f) {
```

Comment:
just FYI I just merged a change that moves this snippet into dartpad_picker_main. I created an issue to track: https://github.com/dart-lang/site-www/issues/2581